pages/docs/reference/whatsnew14.md

```diff
@@ -476,6 +476,92 @@ For more information about default methods in the Java interop, see the [documen
 
 ## Kotlin/Native
 
+### Support for Kotlin’s suspending functions in Swift and Objective-C
+
+In 1.4, we add the basic support for suspending functions in Swift and Objective-C. Now, when you compile a Kotlin module
+into an Apple framework, suspending functions are available in it as functions with callbacks (`completionHandler` in 
+the Swift/Objective-C terminology). When you have such functions in the generated framework’s header, you can call them
+from your Swift or Objective-C code and even override them.
+
+For example, if you write this Kotlin function:
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+suspend fun queryData(id: Int): String = ...
+```
+</div>
+
+…then you can call it from Swift like so:
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```swift
+queryData(id: 17) { result, error in
+   if let e = error {
+       print("ERROR: \(e)")
+   } else {
+       print(result!)
+   }
+}
+```
+</div>
+
+For more information about using suspending functions in Swift and Objective-C, see the [documentation](native/objc_interop.html).
+
+### Objective-C generics support by default
+
+Previous versions of Kotlin provided experimental support for generics in Objective-C interop. Since 1.4, Kotlin/Native 
+generates Apple frameworks with generics from Kotlin code by default. In some cases, this may break existing Objective-C
+or Swift code calling Kotlin frameworks. To have the framework header written without generics, add the `-Xno-objc-generics` compiler option.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+binaries.framework {
+     freeCompilerArgs += "-Xno-objc-generics"
+}
```

Comment:
Such code fragments are known to be very confusing to users, especially when using our CocoaPods integration.
Because this code creates new framework, while CocoaPods integration creates frameworks implicitly, so users end up having two frameworks.

The alternative approach for adding compiler arguments [below](https://github.com/JetBrains/kotlin-web-site/pull/1779/files#diff-cae9fb9cc38414d4c10afcf1b936c58dR552) is preferable.

Reply:
(And yes, I know that we used the same code in blog post and in the documentation, sorry for bringing this consideration late)

Reply:
Updated the example